pkg/host/syscalls_linux.go

```diff
@@ -315,6 +320,7 @@ var syzkallSupport = map[string]func(*prog.Syscall, *prog.Target, string) (bool,
 	"syz_fuse_handle_req":         isSyzFuseSupported,
 	"syz_80211_inject_frame":      isWifiEmulationSupported,
 	"syz_80211_join_ibss":         isWifiEmulationSupported,
+	"syz_nic_vf":                  isNicVFSupported,
```

Comment:
This is checking for support of pseudo-syscalls defined in sys/linux/*.txt files.
You did not define syz_nic_vf syscall, so this is not necessary.

Reply:
Change made.